cloud-checksum/src/task/copy.rs

```diff
@@ -214,7 +214,7 @@ impl CopyTaskBuilder {
         let min_part_size = destination_copy.min_part_size();
 
         // Only use the sums file if the size is not set.
-        let sums = if self.part_size.is_none() {
+        /*let sums = if self.part_size.is_none() {
```

Comment:
This snippet hit some AWS operations that fail without credentials. Not sure which ones though.